dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/ingest/IngestServiceBean.java

```diff
@@ -825,6 +813,8 @@ public boolean ingestAsTabular(Long datafile_id) {
 
             logger.log(Level.WARNING, "Ingest failure.", ingestEx);
             return false;
+        } finally {
+            IOUtils.closeQuietly(inputStream);
```

Comment:
Couldn't we put that stream into try-with-resources?

Reply:
There will be subtle difference in execution: If close will fail then in original code it won't cause the whole ingest to fail but it will be quietly ignored. If we make this with try-with-resources then fail when closing will cause whole ingest to fail. Although I think that your proposition is good. At least we would discover more quickly that we have some issues with closing streams.